src/lightstep-graph/graph.js

```diff
@@ -69,39 +69,55 @@ function graphDirective($rootScope, timeSrv, popoverSrv, contextSrv) {
           return;
         }
 
-        // ignore if we are the emitter
+          // ignore if we are the emitter
         if (!plot || evt.panel.id === panel.id || ctrl.otherPanelInFullscreenMode()) {
           return;
         }
 
-        tooltip.show(evt.pos);
-      }, scope);
+          tooltip.show(evt.pos);
+        },
+        scope
+      );
 
-      appEvents.on('graph-hover-clear', function(event, info) {
-        if (plot) {
-          tooltip.clear(plot);
-        }
-      }, scope);
+      appEvents.on(
+        "graph-hover-clear",
+        function (event, info) {
+          if (plot) {
+            tooltip.clear(plot);
+          }
+        },
+        scope
+      );
 
       function getLegendHeight(panelHeight) {
         if (!panel.legend.show || panel.legend.rightSide) {
           return 0;
         }
 
         if (panel.legend.alignAsTable) {
-          var legendSeries = _.filter(data, function(series) {
+          var legendSeries = _.filter(data, function (series) {
             return series.hideFromLegend(panel.legend) === false;
           });
-          var total = 23 + (21 * legendSeries.length);
-          return Math.min(total, Math.floor(panelHeight/2));
+          var total = 23 + 21 * legendSeries.length;
+          return Math.min(total, Math.floor(panelHeight / 2));
         } else {
           return 26;
         }
       }
 
+      function getHeight() {
+        if (ctrl.height) {
+          return ctrl.height;
+        }
+        if (ctrl.scope.$parent.size.height) {
```

Comment:
This was the main change - on first render, the height property is `undefined` in the `ctrl` object. `$parent` let us access the parent element, and grab the height to calculate dimensions for the graph